### DISCUSSION TITLE
项目模块名称修改后 服务启动不起来

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#203] [02.提问]
我把lamp-cloud 项目名称包括pom文件里的artifactId 都改了，naocos里 对应的lamp-gateway-server.yml里的routes uri也改成了修改后的名称 但是项目启动报着两个错
<img width="929" alt="image" src="https://user-images.githubusercontent.com/6092525/194795820-f5299a3e-26bd-4c18-a46d-a466f27d70e9.png">
<img width="1125" alt="image" src="https://user-images.githubusercontent.com/6092525/194795833-eda4dec5-1aed-42fa-b298-431d6de0f81d.png">
请问还需要修改什么嘛

**Comment by zuihou** [comment]
yml 中的配置，要和代码中的配置一致，确保项目能正确读取yml 中的配置

**Reply by zqb666** [reply]
没事了 解决了 啥naocos中的yml文件名称也要改成对应的名称
